m4_sessions/1711-inning/Group_2.md

```diff
@@ -1,9 +1,19 @@
 Group Two
 
 Session Title: Description: Why should students come to this session?
+Because Turing is designed around failure and how to learn from that failure. This class seeks to teach how to more effectively learn from that struggle.
 
 Session Outcomes: What will students leave with or gain from this session?
+How to see failure as a stepping stone to future successes.
 
 Session Outline: Include the flow of the session and the time for each section.
+15 Minutes - Student introductions and why we're here
+30 Minutes - Presentation regarding struggle
```

Comment:
30 minutes of presentation is a long time.  Consider breaking this up into some presentation, then group discussion/practice, then some more content.